corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -553,6 +553,40 @@ hqDefine("cloudcare/js/formplayer/app", function () {
         $.ajax(options);
     });
 
+    FormplayerFrontend.getChannel().reply("interval_sync-db", function (unusedOptions) {
+        var user = FormplayerFrontend.getChannel().request('currentUser'),
+            username = user.username,
+            domain = user.domain,
+            formplayerUrl = user.formplayer_url,
+            complete,
+            data = {
+                "username": username,
+                "domain": domain,
+                "restoreAs": user.restoreAs,
+                "app_id": unusedOptions.appId,
+            },
+            options;
+
+        complete = function (response) {
+            if (response.status === 'retry') {
+                FormplayerFrontend.trigger('retry', response, function () {
+                    options.data = JSON.stringify($.extend(true, {mustRestore: true}, data));
```

Comment:
Since we're immediately stringifying, is the deep copy through `$.extend` necessary?

Reply:
what do you mean by a deep copy? Doesn't $.extend just merge two objects? I did remove `true` argument since recursive merging isn't needed [c4bd9ed](https://github.com/dimagi/commcare-hq/pull/34161/commits/c4bd9edb57e0d264fddc7160f9cda702ad6f18f6)

Reply:
I see us use `_.extend` more frequently -- there's just no option for deep copying. As far as I am aware it's identical to the updated version here, just underscore instead of jQuery.